src/hotspot/share/utilities/intpow.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
`Copyright (c) 2024, Oracle`? Is there a co-author here?

Reply:
There isn't,  thanks, I'll remove it :+1: